governance-exercises/Adjust Spork D3M Parameters and DAI Interest Rate -- July 24, 2023.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Adjust Spork Protocol D3M Parameters and DAI Interest Rate - July 24, 2023
```

Comment:
Part of the clue you should either be atomizing more, or presenting the vote at a higher abstraction level is when you find yourself using 'and' in the title.